packager/tools/generate_license_notice.py

```diff
@@ -13,34 +13,18 @@
 
 LICENSE_FILES = set(['LICENSE', 'LICENSE.TXT', 'COPYING'])
 PRUNE_PATHS = [
-    # Identical to third_party/icu.
-    os.path.join('base', 'third_party', 'icu'),
-
-    # Comes with Chromium base, not used in packager.
-    os.path.join('base', 'third_party', 'libevent'),
-    os.path.join('base', 'third_party', 'nspr'),
-    os.path.join('base', 'third_party', 'superfasthash'),
-    os.path.join('base', 'third_party', 'xdg_mime'),
-    os.path.join('base', 'third_party', 'xdg_user_dirs'),
-
-    # Used for development and test, not in the binary build.
-    os.path.join('buildtools', 'third_party', 'libc++', 'trunk', 'utils',
-                 'google-benchmark'),
-    os.path.join('testing', 'gmock'),
-    os.path.join('testing', 'gtest'),
-    os.path.join('third_party', 'binutils'),
-    os.path.join('third_party', 'boringssl', 'src', 'third_party',
-                 'googletest'),
-    os.path.join('third_party', 'gold'),
-    os.path.join('tools', 'gyp'),
-
-    # Comes with Boringssl, not used in packager.
-    os.path.join('third_party', 'boringssl', 'src', 'third_party',
-                 'android-cmake'),
+    # Used for testing, not in the binary build.
+    os.path.join('third_party', 'json'),
+    os.path.join('third_party', 'googletest'),
+
+    # The Java side of icu, not used in Packager.
+    os.path.join('third_party', 'icu', 'source', 'icu4j'),
+
+    # Dependencies of dependencies, not actually used in Packager.
+    os.path.join('third_party', 'libpng', 'source', 'contrib'),
```

Comment:
nit: Move it after line 27 so the two icu are grouped together.

Reply:
Done